packs/feats/fledgling-flight.json

```diff
@@ -29,7 +29,33 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Ancestry Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "fledgling-flight",
+<<<<<<< Updated upstream
+=======
+                "predicate": [
+                    {
+                        "not": "self:effect:juvenile-flight"
+                    },
+                    {
+                        "not": "feat:fully-flighted"
+                    }
+                ],
+>>>>>>> Stashed changes
+                "toggleable": true
+            },
+            {
+                "key": "BaseSpeed",
```

Comment:
This is a merge conflict.

This is also deleting several system files that shouldn't be touched, and the familiar focus familiar ability.

It will be best at this point to hard reset your branch and start again, pushing only the files you are actually changing. To hard reset do

```
git checkout master
git reset --hard upstream/master
```
If you can't do that hard reset then your upstream isn't set, in which case run `git remote add upstream https://github.com/foundryvtt/pf2e.git`